app/classes/textile.rb

```diff
@@ -301,36 +315,59 @@ def strip_out_sp_cfr_and_sensu(str)
       sub(/ sp\.$/, "")
   end
 
-  # Textile should be able to create internal links to Glossary Terms
-  # See https://www.pivotaltracker.com/story/show/76993598
-  # Convert _object name_ and _object id_ in a textile string.
-  def check_other_links!
+  # rubocop:disable Style/RegexpLiteral
+  # cop gives false positive
+  OTHER_LINK_PATTERN = /
+    (?<prefix> ^|\W)
+    (?: _+)
+    (?<type>
+      [a-zA-Z]+ # model name
+      (?: _[a-zA-Z]+)? # optionally including underscores
+    )
+    \s+
+    (?<id> [^_\s](?:[^_\n]+[^_\s])?) # id -- integer or string
+    (?: _+)
+
+    (?! (?: \w|<\/[a-z]+>)) # discard if followed by word char or html close tag
```

Comment:
@pellaea, @mo-nathan 
### Is there a better solution?
This line prevents Textile from munging html tags.
Although I don't do it often, I especially wanted to be able to use things like `<code>`.